dem/core/user_output.py

```diff
@@ -0,0 +1,37 @@
+"""The core can provide information through this module to the user."""
+# dem/core/user_output.py
+
+from abc import ABC, abstractmethod
```

Comment:
is it valid module?

Reply:
It is! https://docs.python.org/3/library/abc.html